pyvista/examples/planets.py

```diff
@@ -0,0 +1,917 @@
+"""Downloadable datasets of 3D Celestial Bodies.
+
+Examples
+--------
+>>> from pyvista import examples
+>>> mesh = examples.planets.load_moon()
+>>> mesh.plot()
+
+"""
+
+import numpy as np
+
+import pyvista
+from pyvista import examples
+
+from .downloads import _download_and_read
+
+
+def _sphere_with_texture_map(radius=1.0, lat_resolution=50, lon_resolution=100):
+    """Sphere with texture coordinates.
+
+    Parameters
+    ----------
+    radius : float, default: 1.0
+        Sphere radius.
+
+    lat_resolution : int, default: 100
+        Set the number of points in the latitude direction.
+
+    lon_resolution : int, default: 100
+        Set the number of points in the longitude direction.
```

Comment:
We never specify defaults like this, we always have "optional" (as they were before the change). We usually even make it a point not to mention clear defaults in the variable description, because the defaults are there in the signature. Was there a specific reason why you added these @tkoyama010, or was this just a fluke? :)

Reply:
Actually, there are about 130 instances of this pattern, but I've never come across it!

Reply:
It is just a fluke. Do you have any idea?